client/me/security-2fa-disable/index.jsx

```diff
@@ -132,11 +132,11 @@ class Security2faDisable extends Component {
 			return (
 				<div className="security-2fa-disable__prompt">
 					<FormSectionHeading>
-						{ translate( 'Disable Two-Step Authentication' ) }
+						{ translate( 'Disable two-step authentication' ) }
 					</FormSectionHeading>
 					<p>
 						{ translate(
-							'You are about to disable Two-Step Authentication. ' +
+							'You are about to disable two-step authentication. ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'You are about to disable Two-Step Authentication. This means we will no longer ask for your authentication code when you sign into your {{strong}}%(userlogin)s{{/strong}} account.' )` *ES Score: 10*